imod/mf6/dis.py

```diff
@@ -88,7 +97,7 @@ class StructuredDiscretization(Package, IRegridPackage):
     _regrid_method = {
         "top": (RegridderType.OVERLAP, "mean"),
         "bottom": (RegridderType.OVERLAP, "mean"),
-        "idomain": (RegridderType.OVERLAP, "mode"),
+        "idomain": (RegridderType.OVERLAP, "sum"),
```

Comment:
here, you could add a TODO to remind us to put "mode" back for idomain regridding